dev/tests/integration/testsuite/Magento/Customer/Block/Adminhtml/Edit/Tab/CartTest.php

```diff
@@ -5,32 +5,52 @@
  */
 namespace Magento\Customer\Block\Adminhtml\Edit\Tab;
 
+use Magento\Backend\Block\Template\Context;
+use Magento\Backend\Model\Session\Quote as SessionQuote;
 use Magento\Customer\Controller\RegistryConstants;
+use Magento\Framework\ObjectManagerInterface;
+use Magento\Framework\Registry;
+use Magento\Quote\Model\Quote;
+use Magento\Store\Model\StoreManagerInterface;
 
 /**
  * Magento\Customer\Block\Adminhtml\Edit\Tab\Cart
  *
  * @magentoAppArea adminhtml
+ * @SuppressWarnings(PHPMD.CouplingBetweenObjects)
  */
 class CartTest extends \PHPUnit\Framework\TestCase
 {
     const CUSTOMER_ID_VALUE = 1234;
```

Comment:
Failed to understand for what purpose `CUSTOMER_ID_VALUE` is used for, - but seems OK.
@engcom-Golf , - thank you.